src/masoniteorm/scopes/SoftDeletesMixin.py

```diff
@@ -4,5 +4,10 @@
 class SoftDeletesMixin:
     """Global scope class to add soft deleting to models."""
 
+    __deleted_at__ = "deleted_at"
+
     def boot_SoftDeletesMixin(self, builder):
         builder.set_global_scope(SoftDeleteScope())
```

Comment:
We can pass it here?
```suggestion
        builder.set_global_scope(SoftDeleteScope(column=self.__deleted_at__))
```

?